addons/point_of_sale/models/account_journal.py

```diff
@@ -27,3 +28,17 @@ def _search(self, args, offset=0, limit=None, order=None, count=False, access_ri
     def onchange_type(self):
         if self.type not in ['bank', 'cash']:
             self.journal_user = False
+
+    @api.multi
+    def write(self, vals):
+        for journal in self:
+            if 'active' in vals and self.env['pos.session'].search_count([('state', '!=', 'closed'), '|', ('config_id.journal_id', '=', journal.id), ('config_id.journal_ids', 'in', journal.id)]):
+                raise UserError(_("You cannot delete Journals that are used by active PoS sessions."))
```

Comment:
"you can not delete" but you are making `if 'active' in vals`. Are you sure your check is correct?

Reply:
The check is correct, the string is not. :see_no_evil: